tex/polyglossia.sty

```diff
@@ -236,7 +236,6 @@
       \RLE
     }
   }
-  \ExplSyntaxOff
```

Comment:
I am not sure why this ended up here, but this is not part of the fix to this problem, right?

Reply:
OK I see this is a leftover from 7df891a8c5309f16f7. We should address this in a separate commit.

Reply:
> We should address this in a separate commit.

Done at 56f2da4b2356651d.

Reply:
> I am not sure why this ended up here, but this is not part of the fix to this problem, right?

Yes, sorry, forgot to mention that.